src/main/java/org/marketdesignresearch/mechlib/outcomerules/ccg/paymentrules/EuclideanNorm.java

```diff
@@ -59,7 +59,6 @@ public PrimaryObjective getPrimaryObjectiveConstraint(IMIP program) {
         IMIP newProgram = MIPWrapper.makeMIPWithoutObjective(program);
         addNormObjective(newProgram);
         IMIPResult result = solveProgram(newProgram);
-        getMipInstrumentation().postMIP(MipInstrumentation.MipPurpose.PAYMENT, newProgram, result);
         metaInfo.setLpSolveTime(result.getSolveTime());
```

Comment:
Please double check, but solveProgram already calls preMIP and postMIP so the postMIP here can be removed.

Reply:
double checked, you're right 👍